cultivation/scripts/flashcore/cli/flashcards_cli.py

```
@@ -0,0 +1,56 @@
+DB_PATH = PROJECT_ROOT / "cultivation" / "data" / "flash.db"
+
+# A default, hardcoded user UUID. In a real system, this would come from a config file.
+DEFAULT_USER_UUID = uuid.UUID("00000000-0000-0000-0000-000000000001")
+
+
+@app.command()
+def review(
+    deck_name: str = typer.Argument(..., help="The name of the deck to review."),
+    db_path: Path = typer.Option(
+        DB_PATH, "--db-path", help="Path to the flashcard database file."
+    ),
```

Ruff (0.11.9)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)